docs/src/main/asciidoc/_admin-guide/management-tasks/JVM_settings.adoc

```diff
@@ -79,6 +79,33 @@ heap size.
 }
 ----
 
+=== Using filesystem locations as JVM options on domain mode
+The link:Command_line_parameters.html#controlling-filesystem-locations-with-system-properties[Controlling filesystem locations with system properties] section describes the available system properties associated with relevant WildFly file system paths. In addition to all the domain mode properties, the following server specific properties are also available for resolution as JVM options:
```

Comment:
Hi, the link redirect me to github 404 page.

Reply:
Good catch @jedla97, I saw the link working fine in my IDE but it was not working in the final documentation generated. I've fixed the link and rebased the branch. Thanks!